### DISCUSSION TITLE
TPOT on AWS

### DISCUSSION THREAD

**Discussion by tomquinn8** [telekom-security/tpotce#1033] [General]
I've spent the last couple of days struggling to get TPOT to work on AWS, I decided to give it one last go today and appear to have been successful but only by modifying line 524 of standard.yml to uncomment 'mem_limit: 4g'.

I'm sure there is a reason for it being commented but for anyone else struggling with AWS (large instance size) this could be the solution you're looking for.

**Comment by t3chn0m4g3** [comment]
Thanks, this is really a mixed bag. If I do not limit to `4g` then people usually have problems to get it running on machines with just 8GB of RAM. With 22.x this should be no problem, though, since RAM management of the Elastic Stack has drastically improved with 8.0.0.

**Reply by tomquinn8** [reply]
With 8GB of RAM but without limiting it to 4g I couldn't even SSH to the server it was fully locked up which also prevents the use of --type=user on the install script because then it auto reboots and becomes non-responsive.

**Reply by t3chn0m4g3** [reply]
AWS support is solely community developed, feel free to check the 22.x branch, there were some activities in that regard. It is the dev branch, so dragons are to be expected.

**Comment by nc-sec** [comment]
@tomquinn8 In the mean time a t3.xlarge will suffice. For perspective, I ran tpot on a t3.2xlarge and it was happy to eat up ~20GB RAM.
